lib/cc/engine/flog.rb

```diff
@@ -129,7 +129,7 @@ def issue name, datum, location, score
         end
 
         remediation_points =
-          (BASE_REMEDIATION_POINTS + (OVERAGE_REMEDIATION_POINTS * score)).
+          (BASE_REMEDIATION_POINTS + (OVERAGE_REMEDIATION_POINTS * (score - config["max_score"]))).
```

Comment:
Nice! Glad we're using the diff to inform remediation points. 

WDYT about moving this calculation out into a private method like

```
def remediation_points(score)
  ...
end
```
and just passing into the issue json
?

Works fine either way so take or leave 👍 

#metacomplexityscoring